support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -290,6 +292,28 @@ if (ratePlanDescription?.billingPeriod === 'Annual') {
 	paymentFrequency = 'quarter';
 }
 
+const processPayment = (statusResponse: StatusResponse) => {
+	const { trackingUri, status, failureReason } = statusResponse;
+	const jobId = new URL(trackingUri).searchParams.get('jobId') ?? '';
+	if (status === 'success') {
+		window.location.href = `uk/thank-you?jobId=${jobId}`;
+	} else if (status === 'failure') {
+		console.error(failureReason);
```

Comment:
Is a console log here sufficient error handling, should we be logging this somwhere?

Reply:
Trying to keep the PR small and specific to the handling. I've tracked this [here](https://github.com/guardian/support-frontend/issues/5722).

We will need to take a step back I think and work on setting errors generically across the form.

[That's been tracked here](https://trello.com/c/ihabrisb/802-form-validation-for-generic-checkout).

Reply:
I've used a few `console.error`s in this file to try track the areas we'll need to add the error handling.